core/tests/Test.Microsoft.Identity.Core.Unit/Mocks/MockHelpers.cs

```diff
@@ -315,5 +316,19 @@ public static HttpMessageHandler CreateInstanceDiscoveryMockHandler(string url,
                 }
             };
         }
+
+        public class TestLegacyCachePersistance : ILegacyCachePersistance
+        {
+            private byte[] data;
+            public byte[] LoadCache()
+            {
+                return data;
+            }
+
+            public void WriteCache(byte[] serializedCache)
+            {
+                data = serializedCache;
+            }
+        }
```

Comment:
new file

Reply:
?

Reply:
Please place this class in a new file. See other objects such as the TestLogger.cs

Reply:
ok